src/panels/lovelace/cards/hui-tile-card.ts

```diff
@@ -368,16 +377,18 @@ export class HuiTileCard extends LitElement implements LovelaceCard {
             hasHold: hasAction(this._config!.hold_action),
             hasDoubleClick: hasAction(this._config!.double_tap_action),
           })}
-          role="button"
-          tabindex="0"
+          role=${hasCardAction ? "button" : ""}
+          tabindex=${hasCardAction ? "0" : undefined}
           aria-labelledby="info"
-          @mousedown=${this.handleRippleActivate}
-          @mouseup=${this.handleRippleDeactivate}
-          @mouseenter=${this.handleRippleMouseEnter}
-          @mouseleave=${this.handleRippleMouseLeave}
-          @touchstart=${this.handleRippleActivate}
-          @touchend=${this.handleRippleDeactivate}
-          @touchcancel=${this.handleRippleDeactivate}
+          @mousedown=${hasCardAction ? this.handleRippleActivate : undefined}
+          @mouseup=${hasCardAction ? this.handleRippleDeactivate : undefined}
+          @mouseenter=${hasCardAction ? this.handleRippleMouseEnter : undefined}
+          @mouseleave=${hasCardAction ? this.handleRippleMouseLeave : undefined}
+          @touchstart=${hasCardAction ? this.handleRippleActivate : undefined}
+          @touchend=${hasCardAction ? this.handleRippleDeactivate : undefined}
+          @touchcancel=${hasCardAction
```

Comment:
Probably nicer to handle this in the `handle` function?

Reply:
Not sure I'm seeing the "nicer" approach. 

Do you want to add an additional class member (`hasCardAction`), and then in every handler function do nothing and return if that member is true?